### ISSUE TITLE
Any chance to use an old iOS 9.3.5 iPad?

### ISSUE THREAD

**Issue by MAPEZ1** [EricAndrechek/homebridge-display#118]
We have two old iPad's and it would be great if they could be used with Homebridge-Display as wallmount devices.
Homebridge-Display works on iOS 14 Safari, but the old are 9.3.5
I tried also other browsers on the iPad, but same result (see screenshot)
I know they are old, but a second life for them, instead of a new device, would be great.
Any chance ? 

**config
       {
            "Config": {
                "background": "https://i.postimg.cc/MpjZQcnn/background-image.jpg",
                "app_id": "https://i.postimg.cc/jjCsX4gH/home-black-ios.png",
                "private": false,
                "port": 6969,
                "customTitle": "Mapez Home"
            },
            "Boxes": {
                "Box1": "iot",
                "Box2": "blank",
                "Box3": "weather",
                "Box4": "calculator",
                "Box5": "blank",
                "Box6": "news"
            },
            "Newsfeed": {
                "feed": [
                    "https://some/index.rss"
                ],
                "delay": 60
            },
            "Weather": {
                "units": "C",
                "api_key": "my_key",
                "lat": xxx,
                "lon": yyy
            },
            "platform": "homebridge-display"
        },

**screenshot
![image](https://user-images.githubusercontent.com/76261897/107863578-d0d7a100-6e55-11eb-8092-58761fbdcbfc.png)

**Comment by EricAndrechek** [comment]
I agree this should be something that works, I am actively looking into it but I am having trouble determining what is going wrong. My leading theory is that there is something wrong with WebSocket communications between the iPads and the server. I am not sure specifically what is going wrong with the WebSockets, though. If you are able (you'll need a mac for this) and willing, I would appreciate it if you could follow [these instructions here](https://www.browserstack.com/guide/how-to-inspect-element-on-iphone) to connect your computer to your iPad, then try to load the page on your iPad. Then you'll be able to see in the console in Safari what the errors are. If you could send those to me, I'll hopefully be able to isolate and fix the issue.

**Comment by MAPEZ1** [comment]
Hi, 100% willing and also Mac :-)
Today I’m a bit short in available time, but for sure during the next two - three days I will look into and hopefully can provide you with insights.
Btw. the Homebridge UI X has also a display issue on old devices.
Status, Config and Log display only a small line below the menu bar. The last update (4.38.0) fix an display issue on old browsers when opening via ip address. I gave it a try via raspi.local:xxxx,
but it’s still the same.

**Comment by EricAndrechek** [comment]
Thank you, I appreciate the help and insight about Homebridge UI issues too. No worries about time, just whenever you are free is great!

**Comment by MAPEZ1** [comment]
Using macOS Catalina 10.15.7 and I'm having problems to successfully webinspect the iOS 9.3.5 iPad.
MacBook and iPad where restarted.
The device shows up in the Develop Menu but it's possibly to old to be supported
![image](https://user-images.githubusercontent.com/76261897/107942474-97a14d00-6f8b-11eb-8ce3-bae4956abe9a.png)
At iOS 14 iPad is works
![image](https://user-images.githubusercontent.com/76261897/107942365-6de82600-6f8b-11eb-84f3-20f140f993d7.png)

**Comment by EricAndrechek** [comment]
That's really strange, it seems to be a common error in looking into it further. I used a free trial of BrowserStack to open Safari on an iPad running iOS 9.3.5, and it seemed to be working fine. I will continue to look into this and will try and add some additional debugging features into my next update to help troubleshoot further. I will be leaving this issue open until I find a definitive answer as to what is happening. Apologies for the fact that it isn't working yet, I can understand the annoyance.
